.github/workflows/test.yml

```diff
@@ -1,45 +0,0 @@
-name: test
-
-on:
-  pull_request:
-    branches:
-      - main
-      - develop
```

Comment:
@tischi  do you follow a dev pattern where PRs to `develop` are common? Then `- develop` should also be kept in the `build.yml` file added in this PR.

Reply:
Yes.